src/views/dependencyExplorer.ts

```diff
@@ -90,6 +90,23 @@ export class DependencyExplorer implements Disposable {
 
                 this.reveal(uri, false /*force to reveal even the sync setting is turned off*/);
             }),
+            instrumentOperationAsVsCodeCommand(Commands.JAVA_PROJECT_EXPLORER_CONFIGURE_FILTERS, async () => {
+                const filters = await window.showQuickPick(
+                    [{
+                        label: "Non-Java resources",
+                        picked: Settings.nonJavaResourcesFiltered(),
+                    }],
+                    {
+                        placeHolder: "Select filters to apply to the Java Projects Explorer",
```

Comment:
add `ignoreFocusOut`? since users may have chance to check the files in the explorer.

Reply:
Ok, added